docs/dev/ui_description.rst

```diff
@@ -268,7 +268,7 @@ implemented inside a widget:
 Glue callback properties
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-In cases where trailets are not apppropriate (e.g. where some python object is
+In cases where trailets are not appropriate (e.g. where some python object is
```

Comment:
Are you sure this wasn't a pun...? `app`propriate? 😉

Reply:
You may as well fix "trailets" to "traitlets" while you're at it 😆

Reply:
And now having looked at that doc page...it's actually wrong throughout the whole page! I had to check https://traitlets.readthedocs.io/en/stable/ and make sure I wasn't remembering the name wrong.

Reply:
Is putting comma after `e.g.` out of scope? 👀

Reply:
trailets -> traitlets is done.  There were several instances.

Reply:
commas added after `.e.g.` too (there were many other instances). :)